src/docks/playlistdock.cpp

```diff
@@ -1113,3 +1113,34 @@ void PlaylistDock::onProducerModified()
     // The clip name may have changed.
     emitDataChanged(QVector<int>() << PlaylistModel::FIELD_RESOURCE);
 }
+
+void PlaylistDock::on_addFilesToPlaylistButton2_clicked()
+{
+    on_addFilesToPlaylistButton_clicked();
+}
+
+void PlaylistDock::on_addFilesToPlaylistButton_clicked()
+{
+    QMimeData* mimeData = new QMimeData;
```

Comment:
This `mimeData` is not destroyed and leaked because `onDropped` does not. It is probably OK to simply declare this as  `QMimeData mimeData;` and then pass it like `onDropped(&mimeData, ...)`

Reply:
OK, I will make the change.